tools/check_yaml

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash -e
+
+success=1
+YAMLS=$(git ls-files schedule/ test_data/ | grep '.ya\?ml$' | xargs echo)
+
+if test -n "$YAMLS"; then \
```

Comment:
```suggestion
if test -n "$YAMLS"; then
```
If this is in an external bash file, then all the `\` at the end can be omitted.

Same applies to almost every line in this script.